### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mallorydxw**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,61 +1,107 @@
...
+  - the default user is `admin` with password `admin`
+  - will also install [whippet](https://github.com/dxw/whippet) dependencies
+  - setup configuration files are in the `setup/` directory
+- `script/update` - run any updates after checking out the latest version
+  - this will update whippet dependencies to the latest versions
+- `script/console` - open a `/bin/sh` console inside the main continer
+- `script/bootstrap` - update dependencies (this is called by `script/setup` and `script/update`)
+- `bin/wp` - run [wp-cli](https://wp-cli.org/)
+- `docker-compose.yml` - the Docker Compose configuration
+- `config/server.php` - configuration that would normally go in `wp-config.php` (checked into the repository)
```
Comment (RobjS) [inline README.md]: Would it be worth clarifying that even though checked in, this only runs within the dev container, not on the staging or prod environments?

File: README.md [hunk 2]
```diff
@@ -1,61 +1,107 @@
...
+  - WordPress will be running on http://localhost
+  - MailCatcher will be running on http://localhost:1080
+  - Beanstalk Console will be running on http://localhost:2080
+- `script/setup` - install WordPress
+  - the default user is `admin` with password `admin`
+  - will also install [whippet](https://github.com/dxw/whippet) dependencies
+  - setup configuration files are in the `setup/` directory
+- `script/update` - run any updates after checking out the latest version
+  - this will update whippet dependencies to the latest versions
+- `script/console` - open a `/bin/sh` console inside the main continer
```
Comment (RobjS) [inline README.md]: Minor typo: "continer" => "container"

File: README.md [hunk 3]
```diff
@@ -1,61 +1,107 @@
...
+- `bin/wp` - run [wp-cli](https://wp-cli.org/)
+- `docker-compose.yml` - the Docker Compose configuration
+- `config/server.php` - configuration that would normally go in `wp-config.php` (checked into the repository)
+- `config/server-local.php` - configuration that would normally go in `wp-config.php` (not checked into the repository)
+- `setup/internal.sh` - this is run by `script/setup`, it runs the wp-cli commands to install WordPress
+
+## Requirements
+
+- [Docker](https://docs.docker.com/docker-for-mac/install/)
+- [Docker Compose](https://docs.docker.com/compose/install/)
```
Comment (RobjS) [inline README.md]: The Docker Desktop installation now includes Docker Compose. So this wouldn't be needed if they'd installed Docker via the previous link.

**Review by RobjS — CHANGES_REQUESTED** [review]

**Comment by mallorydxw** [comment]
@RobjS Thanks! Resolved all three. Ready for another review.

**Review by RobjS — APPROVED** [review]

**Pull request merged**